### PR TITLE
pc_helper: Add tests and cleanup unused code

### PR DESCRIPTION
**Pull request by asmorodskyi**
PR contains several commits :
-   Add ability allowing to check Public Cloud related logic outside of bot-ng
-   Drop function because it is not used anymore
-   Drop use of variable PUBLIC_CLOUD_PINT_QUERY    
-   Add tests for pc_helper
-   Switch to pythonic documentation format

### CODE REVIEW

**Review by Martchus — APPROVED** [review]
I guess you know what you're doing :-)
(The style-check is failing, though.)

File: openqabot/pc_helper.py
Comment (pdostal) [inline openqabot/pc_helper.py:55]: But what about the others?

File: openqabot/pc_helper.py
Comment (ricardobranco777) [inline openqabot/pc_helper.py:81]:
```suggestion
            log_error += f' for {settings["PUBLIC_CLOUD_PINT_NAME"]}'
```

Add a space to get a proper error message.

**Review by ricardobranco777 — APPROVED** [review]
Missing space for error message.

LGTM

**Review by okurz — CHANGES_REQUESTED** [review]
See https://github.com/openSUSE/qem-bot/actions/runs/5324473912/jobs/9643874264?pr=121#step:6:12

File: openqabot/pc_helper.py
Comment (asmorodskyi) [reply openqabot/pc_helper.py:55]: somehow I was under impression that we using `  PUBLIC_CLOUD_PINT_QUERY:`  **only** for Amazon . after double-checking I find out that we also using it for google . thanks for hint I will correct this

Comment (asmorodskyi) [reply openqabot/pc_helper.py]: So I returned `PUBLIC_CLOUD_PINT_QUERY` back . seems like we can not avoid use of something defining provider and in that case it does not matter if it will be just "google"/"amazon" or just full url

**Comment by okurz** [comment]
@asmorodskyi style checks are still failing, please see https://github.com/openSUSE/qem-bot/actions/runs/5331833618/jobs/9660364775?pr=121#step:6:12 and only push a further update after you fixed the style

**Comment by okurz** [comment]
ok, I will not approve further CI jobs until you have explicitly commented "Yes, I have properly formatted the style, python black passes checks locally."

**Comment by asmorodskyi** [comment]
> ok, I will not approve further CI jobs until you have explicitly commented "Yes, I have properly formatted the style, python black passes checks locally."

may I ask why you manually approving CI jobs ? doesn't it break concept of automatic pipeline ? :thinking:

**Comment by asmorodskyi** [comment]
black is passing locally

**Comment by okurz** [comment]
please say "Yes, I have properly formatted the style, python black passes checks locally." :)

> > ok, I will not approve further CI jobs until you have explicitly commented "Yes, I have properly formatted the style, python black passes checks locally."
> 
> may I ask why you manually approving CI jobs ? doesn't it break concept of automatic pipeline ? thinking

No, I am just approving that a CI job may run due to github by default requiring an approval from a maintainer for a new contributor.

File: openqabot/pc_helper.py
Comment (okurz) [inline openqabot/pc_helper.py:16]:
```suggestion
    'publiccloud_tools_<BUILD NUM>.qcow2' is a generic name for an image used by Public Cloud tests to run
    in openQA. A query is supposed to look like this "https://openqa.suse.de/group_overview/276.json" to get
    a value for <BUILD NUM>
```

Comment (asmorodskyi) [reply openqabot/pc_helper.py]: done

**Comment by asmorodskyi** [comment]
black is passing locally , can you please trigger pipeline one more time ?

**Review by okurz — APPROVED** [review]

**Comment by Martchus** [comment]
Looks like this leads to failures in production, e.g. https://gitlab.suse.de/qa-maintenance/bot-ng/-/jobs/1647503/raw and https://gitlab.suse.de/qa-maintenance/bot-ng/-/jobs/1647282/raw.

**Comment by okurz** [comment]
I merged a revert https://github.com/openSUSE/qem-bot/pull/122 for now

**Pull request merged**
